### DISCUSSION TITLE
Change color ToggleSwitch when IsMouseOver=True and IsOn=True

### DISCUSSION THREAD

**Discussion by recetarioMDWiki** [MahApps/MahApps.Metro#4111] [Q&A] [answered]
Hello!

I'm trying to reproduce in MahApps 2.4.5 a style that I modified in MahApps 1.6.5.

The old style looks like this (explanation below):
![image](https://user-images.githubusercontent.com/50943477/119355329-6d0b1280-bca5-11eb-92b1-3e0f18ae557e.png)

1. All white when IsOn = False. The ToggleSwitch control is inside of a Expander Header.
2. Gray5 when IsOn = False and IsMouseOver = True
3. IsOn = True
4. IsOn = True y IsMouseOver = True

In MahApps 2.4.5 I reach all but not 4. I've tried to add a MultiTrigger like this:

```
<MultiTrigger>
    <MultiTrigger.Conditions>
        <Condition Property="IsMouseOver" Value="True" />
        <Condition Property="IsOn" Value="True" />
    </MultiTrigger.Conditions>
    <Setter TargetName="OuterBorder" Property="Fill" Value="{DynamicResource MahApps.Brushes.Gray5}" />
    <Setter TargetName="OuterBorder" Property="Stroke" Value="{DynamicResource MahApps.Brushes.Gray5}" />
</MultiTrigger>
```

But it doesn't work.

Someone know how can I do it?
Thank you so much!

**Comment by timunie** [comment, marked as answer]
Hi @recetarioMDWiki 

just override the needed Brushes in your `Resources`-Section. 

Available Brushes: https://github.com/MahApps/MahApps.Metro/blob/a636cc8295742a230682ad7c5033cf4520b0916c/src/MahApps.Metro/Styles/Themes/Theme.Template.xaml#L485-L517

Usage: 
```xaml
<SolidColorBrrush (OR ANY OTHER BRUSH) x:Key="[THE KEY TO OVERRIDE]" Color="[THE NEW COLOR]" />
```

Example
```xaml
<SolidColorBrrush x:Key="MahApps.Brushes.ToggleSwitch.StrokeOff" Color="White" />
```

Happy coding
Tim

**Reply by recetarioMDWiki** [reply]
Thank you so much, @timunie ! 🎉

That's works perfect for me. I leave here my code as example for anyone that may need it in the future:

```
<Controls:ToggleSwitch OffContent="Example off"
	               OnContent="Example on"
                       Foreground="White">
    <Controls:ToggleSwitch.Resources>
        <SolidColorBrush x:Key="MahApps.Brushes.ToggleSwitch.StrokeOff" Color="White" />
        <SolidColorBrush x:Key="MahApps.Brushes.ToggleSwitch.KnobFillOff" Color="White" />
        <SolidColorBrush x:Key="MahApps.Brushes.ToggleSwitch.StrokeOffPointerOver" Color="{DynamicResource MahApps.Colors.Gray5}" />
        <SolidColorBrush x:Key="MahApps.Brushes.ToggleSwitch.KnobFillOffPointerOver" Color="{DynamicResource MahApps.Colors.Gray5}" />
        <SolidColorBrush x:Key="MahApps.Brushes.ToggleSwitch.FillOnPointerOver" Color="{DynamicResource MahApps.Colors.Gray5}" />
        <SolidColorBrush x:Key="MahApps.Brushes.ToggleSwitch.StrokeOnPointerOver" Color="{DynamicResource MahApps.Colors.Gray5}" />
    </Controls:ToggleSwitch.Resources>
</Controls:ToggleSwitch>
```

Thanks!
Ana.
